src/RealMeService.php

```diff
@@ -453,12 +453,8 @@ public function enforceLogin(HTTPRequest $request, $backUrl = null)
             $backUrl = $this->getBackURL($request);
         }
 
-        $backURL = $this->validSiteURL($backURL);
-        
-        if (!$backUrl) {
```

Comment:
Why is this part being removed? Was it just added there to bypass problems cause by the bad variable name?

Reply:
Hrm, I'm not sure we want to remove this actually - this is forcing some kind of valid back URL. `validSiteURL` should fail to return anything if the URL isn't valid (e.g. is not a URL on this website) - to avoid people phishing access by back-linking to a malicious site.

Reply:
@andreaspiening could you add this back in please?